app/assets/stylesheets/blacklight/_search_results.scss

```diff
@@ -13,7 +13,7 @@
     // bookmarks checkbox on index, give it some
     // lower margin when it collapses
     .index-document-functions {
-      margin-bottom: ($line-height-base / 2);
```

Comment:
I suspect we want to use `$spacing`  and its variants here, instead of hard-coding a rem value?

Reply:
Is spacing in terms of rem values? That is another option. You will see some hard-coded rem values in Bootstrap 4 variables file for spacing/padding though, it's not unheard of. 

Using $spacing is a way of making sure you are using consistent spacing values -- but if we just want to make it consistent with what was in earlier bootstrap 3 versions of blacklight (where things may NOT have been consistent with each other) without setting up that whole "$spacing regime")...

I think in this case using hard-coded rem may be legit, fixing the bug that was there before without making anything worse than it was before.

Reply:
Ok I converted the rem values to functions of $spacer. $spacer defaults to 0.5rem.